app/src/main/java/com/github/se/travelpouch/model/profile/ProfileRepositoryFirebase.kt

```diff
@@ -217,6 +217,46 @@ class ProfileRepositoryFirebase(private val db: FirebaseFirestore) : ProfileRepo
         .addOnFailureListener { onFailure(Exception("getting friend profile failed")) }
   }
 
+  override fun removeFriend(
```

Comment:
Please add some minimal documentation for this function

Reply:
Done